packages/request-node/src/requestNode.ts

```diff
@@ -15,11 +15,15 @@ import ipfsAdd from './request/ipfsAdd';
 import PersistTransaction from './request/persistTransaction';
 import { getEthereumStorage } from './storageUtils';
 
+const packageJson = require('../package.json');
+
 const NOT_FOUND_MESSAGE =
   'Not found\nAvailable endpoints:\n/POST persistTransaction\n/GET getTransactionsByChannelId\n/GET getChannelsByTopic\n/POST /ipfsAdd\nGET getConfirmedTransaction';
 
 const NOT_INITIALIZED_MESSAGE = 'The node is not initialized';
 
+const REQUEST_NODE_VERSION_HEADER = 'Request-Node-Version';
```

Comment:
Most non-standard headers start with `X-` for historical reasons. We should probably call it `X-Request-Node-Version`